react/features/reactions/functions.any.js

```diff
@@ -9,3 +15,74 @@
 export function getReactionsQueue(state: Object) {
     return state['features/reactions'].queue;
 }
+
+/**
+ * Returns reaction key from the reaction message.
+ *
+ * @param {string} message - The reaction message.
+ * @returns {string}
+ */
+export function getReactionKeyByMessage(message: string) {
+    return Object.keys(REACTIONS).filter(key => REACTIONS[key].message === `:${message}:`)[0];
```

Comment:
Maybe use `find` instead of `filter` here so you won't need to fully traverse the array every time 🤔